<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 44, 54);
color:white;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>FreeSerif</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(39, 44, 54);
color:white;</string>
     </property>
     <property name="title">
      <string>SOMME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>351</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="frame1_layout"/>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>170</y>
        <width>341</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="frame2_layout"/>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>170</y>
        <width>351</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="somme_layout"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>214</width>
        <height>38</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">.QPushButton
{
border-style: outset;
border-width: 2px;
border-radius: 5px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: red;
color:white 
}
QPushButton:pressed{ background-color: blue; }</string>
         </property>
         <property name="text">
          <string>FERMER</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">.QPushButton
{
border-style: outset;
border-width: 2px;
border-radius: 5px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: red;
color:white 
}
QPushButton:pressed{ background-color: blue; }</string>
         </property>
         <property name="text">
          <string>SOMME</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>961</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>800</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLabel:hover {
	border: 2px solid rgb(64, 71, 88);
}
QLabel:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
         </property>
         <property name="text">
          <string>CHEMIN DE L'IMAGE_2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_12">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border: 2px solid rgb(52, 59, 72);
	border-radius: 5px;	
	background-color: rgb(52, 59, 72);
}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}</string>
         </property>
         <property name="text">
          <string>IMPORTER IMAGE_2</string>
         </property>
         <property name="icon">
          <iconset resource="files.qrc">
           <normaloff>:/root/icons/folder.png</normaloff>:/root/icons/folder.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>961</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_9">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>800</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLabel:hover {
	border: 2px solid rgb(64, 71, 88);
}
QLabel:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
         </property>
         <property name="text">
          <string>CHEMIN DE L'IMAGE_1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_13">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border: 2px solid rgb(52, 59, 72);
	border-radius: 5px;	
	background-color: rgb(52, 59, 72);
}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}</string>
         </property>
         <property name="text">
          <string>IMPORTER IMAGE_1</string>
         </property>
         <property name="icon">
          <iconset resource="files.qrc">
           <normaloff>:/root/icons/folder.png</normaloff>:/root/icons/folder.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="files.qrc"/>
 </resources>
 <connections/>
</ui>
